CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v0.18.0
 
+- [#51](https://github.com/SuperGoodSoft/solidus_taxjar/pull/51) Add nexus regions method to API
+
+## v0.18.0
```

Comment:
We have v0.18.0 listed twice now. Should we update the top one to say something like `In Development` or `Unreleased`?

Reply:
Woops this is in the wrong place, I'll move it.